package.json

```diff
@@ -7,12 +7,14 @@
     "client"
   ],
   "scripts": {
-    "backend": "cd api && npm run start",
-    "backend-dev": "cd api && npm run server-dev",
-    "frontend": "cd client && npm run build",
-    "frontend-dev": "cd client && npm run dev",
-    "frontend-test": "cd client && npm run test",
-    "test": "jest",
+    "install": "node config/install.js",
+    "upgrade": "node config/upgrade.js",
+    "backend": "NODE_ENV=production node api/server/index.js",
+    "backend-dev": "NODE_ENV=dev npx nodemon api/server/index.js",
+    "frontend": "NODE_ENV=production cd client && npm run build",
+    "frontend-dev": "NODE_ENV=dev cd client && npm run dev",
+    "frontend-test": "NODE_ENV=test cd client && npm run test",
```

Comment:
I thought I changed this and moved the env set to the other package.json :/ I'm not sure if this will work. I'll have a quick look in an hour or so

Reply:
fixed